index.js

```diff
@@ -68,6 +68,32 @@ MongoClient.connect('mongodb://localhost/apella', (err, db) => {
         })
       })
   })
+
+  app.get('/bills-by-rep/:repId', ({ params: { repId } }, res) => {
+    superagent
+      .get(
+        `https://api.propublica.org/congress/v1/members/${repId}/bills/introduced.json`
+      )
+      .set('x-api-key', process.env.PP_Key)
+      .then((resp, err) => {
+        if (err) {
+          return res.sendStatus(500)
+        } else {
+          res.status(200).send(resp.body.results[0].bills)
+        }
+      })
+  })
+
+  app.get('/get-rep-by-id/:repId', ({ params: repId }, res) => {
+    representatives.findOne({ id: repId.repId }, (err, resp) => {
```

Comment:
repId is an object with a repId property?

Reply:
Yes, I needed only the property value.

Reply:
Ah, your renaming params to repId. I think what you want it `{ params: { repId }}`.  Then you can do `findOne({ id: repId})`.

Reply:
This is done